ContributionGuidelines.md

```diff
@@ -0,0 +1,140 @@
+# [DOC2PEN](https://doc2pen.smaranjitghose.codes/)📝
+#### Doc2pen is an Open Source Software that converts your Typed Docs into Handwritten Docs.
+
+# Contribution Guidelines [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues) 🤝🏽🍀:
+
+## File Structure of the Project 🗃️📂
+
+### [Public Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/public) 🧮✨
+This folder contains the stylesheets, images and fonts. Any files within this directory will not be processed by Webpack but copied directly to the build folder.
+
+Files/Folders that are directly located inside the Public Folder: 📥
+- favicon.ico
+- images
+- index.html
+- logo.png
+- logo192.png
+- logo512.png
+- manifest.json
+- _redirects
+- robots.txt
+
+
+### [Src Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/src) 🧮✨
+This folder contains all the main source code for the React application.
+
+Files/Folders that are directly located inside the Src Folder: 📥
+- app.js 
+- app.test.js
+- assets
+- components
+- fonts
+- index.js 
+- index.css 
+- logo.svg
+- pages
+- reportWebVitals.js
+- seo
+- setupTests.js
+
+### [Asset Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/assets) 🧮✨
+This folder contains all the community brand assets
+
+Files/Folders that are directly located inside the Src Folder: 📥
+- .DS_Store
+- images
+- js
+
+### [Readme_Assets Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/readme_assets)🧮✨
+This folder contains all the images used in Readme Markdown.
+
+## To Get the Project on your local machine 💻🧑‍💻👩‍💻
+- Install [Git](https://git-scm.com/downloads) 📥
+- Setup [Github](https://github.com/) Account 📇
+- Fork [this](https://github.com/smaranjitghose/doc2pen) Project 🍴
+<p align = "center">
+  <img src="https://github.com/kanishkaa24/doc2pen/blob/master/readme_assets/Fork%20Project.png?raw=true" width="500" height="200"/>
+</p>
+
+- Clone your forked copy of the project 🧩.
+
+   ```
+   git clone https://github.com/<your_user_name>/doc2pen.git
+   ```
+
+- Navigate to the project directory 📁.
+
+   ```
+   cd doc2pen
+   ```
+
+- Add a reference(remote) to the original repository.
+
+   ```
+   git remote add upstream https://github.com/smaranjitghose/doc2pen.git
+   ```
+
+- Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+   ```
+   git pull upstream master
+   ```
+
+<p align="center">
+  <img src="https://github.com/kanishkaa24/doc2pen/blob/master/readme_assets/Cloning%20Project%20(1).png?raw=true"/>
+</p>
+
+## To Run the Project on your Local Machine ⌨️🖥️🖱️
+- Navigate to the project.
+- Run `npm install`.
```

Comment:
Write the relevance of the command please😅 followed by the command itself within a code block